src/js/utils/fn.js

```diff
@@ -32,9 +32,7 @@ export const bind = function(context, fn, uid) {
   }
 
   // Create the new function that changes the context
-  const bound = function() {
-    return fn.apply(context, arguments);
-  };
+  const bound = fn.bind(context);
```

Comment:
Use the native bind function

Reply:
ooh, this likely will be super helpful in debugging.

Reply:
Yup, it removes an extra stack frame. Goes from data dispatcher directly to the method instead of through this bound method.